src/test/java/uk/gov/pay/ledger/transaction/resource/TransactionResourceIT.java

```diff
@@ -597,4 +597,33 @@ public void getTransactionsForTransactionShouldReturnHttpNotFoundIfParentTransca
                 .statusCode(Response.Status.NOT_FOUND.getStatusCode())
                 .body("message", is("Transaction with id [some-parent-external-id] not found"));
     }
+
+    @Test
+    public void testResourceCSV() {
+        String targetGatewayAccountId = "123";
+        String otherGatewayAccountId = "456";
+
+        aTransactionFixture()
+                .withTransactionType("PAYMENT")
+                .withState(TransactionState.SUBMITTED)
+                .withGatewayAccountId(targetGatewayAccountId)
+                .insert(rule.getJdbi());
+
+        aTransactionFixture()
+                .withTransactionType("PAYMENT")
+                .withState(TransactionState.SUBMITTED)
+                .withGatewayAccountId(otherGatewayAccountId)
+                .insert(rule.getJdbi());
+
+        given().port(port)
```

Comment:
Is it possible to add more assertions ? for headers and data. Not necessarily in this PR